cmake/modules/LXQtTranslateTs.cmake

```diff
@@ -103,43 +87,16 @@ function(lxqt_translate_ts qmFiles)
         set(templateFile "${TR_TRANSLATION_DIR}/${TR_TEMPLATE}.ts")
     endif ()
 
-    if(TR_USE_QT5)
-        # Qt5
-        if (TR_UPDATE_TRANSLATIONS)
-            qt5_patched_create_translation(QMS
-                ${TR_SOURCES}
-                ${templateFile}
-                OPTIONS ${TR_UPDATE_OPTIONS}
-            )
-            qt5_patched_create_translation(QM
-                ${TR_SOURCES}
-                ${tsFiles}
-                OPTIONS ${TR_UPDATE_OPTIONS}
-            )
-        else()
-            qt5_patched_add_translation(QM ${tsFiles})
-        endif()
-    else()
-        # Qt4
-        if(TR_UPDATE_TRANSLATIONS)
-            qt4_create_translation(QMS
-                ${TR_SOURCES}
-                ${templateFile}
-                OPTIONS ${TR_UPDATE_OPTIONS}
-            )
-            qt4_create_translation(QM
-                ${TR_SOURCES}
-                ${tsFiles}
-                OPTIONS ${TR_UPDATE_OPTIONS}
-            )
-        else()
-            qt4_add_translation(QM ${tsFiles})
-        endif()
-    endif()
-
-    if(TR_UPDATE_TRANSLATIONS)
-        add_custom_target("update_${TR_TEMPLATE}_ts" ALL DEPENDS ${QMS})
+    if (TR_UPDATE_TRANSLATIONS)
+        message("Running lxqt-transupdate: ")
+        execute_process(
+            COMMAND lxqt-transupdate
```

Comment:
IMO we should change the logic of getting sources and ts files in `lxqt-transupdate` and use here the `${TR_SOURCES}` and `${tsFiles}` .... so the configuration is exactly on one place (not in `CMakeLists.txt` together with `.translation-update`)